build_tarballs.jl

```diff
@@ -49,7 +49,7 @@ elif [[ "${target}" == x86_64-w64-mingw32 ]]; then
 elif [[ "${target}" == x86_64-unknown-* ]]; then
```

Comment:
I would compare against `*freebsd*`, not `x86_64-unknown-*`, as that could match many non-FreeBSD things.

Reply:
Yep, I agree, I realised that after the PR.